.github/workflows/ci.yml

```diff
@@ -34,10 +34,17 @@ jobs:
     - name: Tests
       run: |
         pip install -r test-requirements.txt
-        py.test pymemcache/test/ -m unit,integration --port ${{ job.services.memcached.ports[11211] }}
+        py.test pymemcache/test/ \
+            -m unit,integration \
+            --port ${{ job.services.memcached.ports[11211] }} \
+            --tls-port ${{ job.services.tls_memcached.ports[11211] }}
 
     services:
       memcached:
         image: memcached:latest
         ports:
         - 11211/tcp
+      tls_memcached:
+        image: scoriacorp/tls_memcached:latest
+        ports:
+        - 11211/tcp
```

Comment:
Do you want to run this one `11212`, which is the default you have listed for the tests?

Reply:
11211 is inside the container. When I'm testing locally I usually do

```
docker run --rm -p 11212:11211 scoriacorp/tls_memcached
```

cause the non-TLS memcache will be already running on port 11211.